docs/releases/v2.0.0.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: -2
 ---
 
-# CSS v2.0.0
+# CSS v3.0.0
```

Comment:
The file name shows a different version @tzhang200

Reply:
Yes, I found I have the version wrong, supposed to be 3.0. I didn't rename the file though, but the version should be shown as 3.0 at the side bar.

Reply:
Changed file name to 3.0 as well.